variables.tf

```diff
@@ -178,3 +178,13 @@ variable "tags" {
   description = "A map of AWS Tags to attach to each resource created"
   default     = {}
 }
+variable "lb_logging_enabled" {
+  type        = bool
+  description = "Option to enable logging of load balancer requests."
+  default     = false
+}
```

Comment:
(nit) would it be better to derive the enabled flag to be true if the user just provides the lb_logging_bucket_name? 🤔 I'm fine either way, I just know I've tried to deduce as much as I can

Reply:
It would, but I assumed that if we passed in an empty string, we'd get some kind of error similar too `Bucket "" does not exists` when creating the ALB.

Reply:
We can check if the lb bucket variable is not equal to `""` and then assume that the user wants lb logging enabled. 🤷 but at the same time I can see this being something a user may want to enable and then disable at will. So on second thought the way you have it is great.